static/app/types/hooks.tsx

```diff
@@ -185,6 +188,7 @@ export type ComponentHooks = {
   'component:codecov-integration-settings-link': () => React.ComponentType<CodecovLinkProps>;
   'component:confirm-account-close': () => React.ComponentType<AttemptCloseAttemptProps>;
   'component:crons-list-page-header': () => React.ComponentType<CronsBillingBannerProps>;
+  'component:crons-onboarding-panel': () => React.ComponentType<CronsOnboardingPanelProps>;
```

Comment:
Looks like we're using the world `monitor` down below. Oops 💀

Reply:
dangg ittt, i guess anytime we refer to actual 'monitors' we call them as such, but the product pages we end up calling 'crons'